scripts/collect_images_from_gaze_pose.py

```diff
@@ -0,0 +1,204 @@
+                                         block_until_arm_arrives, blocking_stand)
+
+from conq.hand_motion import hand_pose_cmd
+from conq.manipulation import blocking_arm_command
+
+from pathlib import Path
+from PIL import Image
+
+import bosdyn.client
+import bosdyn.client.util
+from bosdyn.client.image import ImageClient
+from conq.cameras_utils import get_color_img, RGB_SOURCES, DEPTH_SOURCES
+from conq.clients import Clients
+
+
+# FIXME: To be ignored during push (Stanley's code to be used only for image collection)
+def move_to(clients: Clients, pose, duration = 1, follow=False):
+    """
+    Move the arm to a pose relative to the body
+
+    Args:
+        clients: Clients
+        x: x position in meters in front of the body center
+        y: y position in meters to the left of the body center
+        z: z position in meters above the body center
+        roll: roll in radians
+        pitch: pitch in radians
+        yaw: yaw in radians
+        duration: duration in seconds
+
+    Adapted from conq.manipulation:hand_pose_cmd
+    """
+    
+    try:
+        print("Arm command received")
+        x,y,z,pitch,roll,yaw = pose
+        arm_cmd = hand_pose_cmd(clients, x,y,z,roll,pitch,yaw,duration)
+        
+        blocking_arm_command(clients, arm_cmd)
+        print("Arm command done")
+        return True
+    except Exception as e:
+        print(e)
+        return False
+    
+
+def open_gripper(clients: Clients):
+    try:
+        clients.command.robot_command(RobotCommandBuilder.claw_gripper_open_command())
+        time.sleep(1)  # FIXME: how to block on a gripper command?
+        return True
+    except: return False
+
+
+def verify_estop(robot):
+    """Verify the robot is not estopped"""
+
+    client = robot.ensure_client(EstopClient.default_service_name)
+    if client.get_status().stop_level != estop_pb2.ESTOP_LEVEL_NONE:
+        error_message = 'Robot is estopped. Please use an external E-Stop client, such as the' \
+                        ' estop SDK example, to configure E-Stop.'
+        robot.logger.error(error_message)
+        raise Exception(error_message)
+
+
+def main(argv):
+    # Parse args
+    parser = argparse.ArgumentParser(description="Collect images of Garden tools from Conq's perspective")
+    bosdyn.client.util.add_base_arguments(parser)
+    
+    parser.add_argument('--to-depth',
+                        help='Convert to the depth frame. Default is convert to visual.',
+                        action='store_true')
+    
+    parser.add_argument('--camera', help='Camera to acquire image from.', default='hand',\
+                        choices=['frontleft', 'frontright', 'left', 'right', 'back','hand',
+                        ])
+    parser.add_argument('--auto-rotate', help='rotate right and front images to be upright',
+                        action='store_true')
+
+    parser.add_argument('--gaze-x', required=False, type=float, default=0.75, help='X position for gaze pose')
```

Comment:
`required=False` is not needed here. Any argument that starts with `--` is treated as optional by default.

Reply:
Fixed. Also, this script is omitted from this PR.